src/AutoRest.CSharp/Mgmt/Output/VirtualResource.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using AutoRest.CSharp.Input;
+using AutoRest.CSharp.Mgmt.AutoRest;
+using AutoRest.CSharp.Mgmt.Decorator;
+using AutoRest.CSharp.Mgmt.Models;
+using AutoRest.CSharp.Output.Models;
+
+namespace AutoRest.CSharp.Mgmt.Output
+{
+    /// <summary>
+    /// A virtual resource stands for a resource from another SDK, and it plays a role of anchor of some operations that belong to this resource in another SDK
+    /// </summary>
+    internal class VirtualResource : Resource
```

Comment:
Can we call this a `PartialResource`.  Initially I was reviewing this in the context of how a publisher or location is a virtual resource since its not really an entity that you can create or manage but it is an organizational tool to group and find things in the same group.

The concept here is that this will act like partial class across libraries.

Reply:
I renamed this to PartialResource